src/server/api/controllers/private-channel.controller.js

```diff
@@ -0,0 +1,17 @@
+const knex = require('../../config/db');
+// const Error = require('../lib/utils/http-error');
+// const moment = require('moment-timezone');
+
+const checkPrivateChannel = async (userId1, userId2) => {
```

Comment:
This needs to support multiple users, what if there were 5 users that i wanted to check fx